Snake/BusinessLayer/maze.cs

```diff
@@ -1,59 +1,198 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+
 
 namespace BusinessLayer
 {
-     public class maze
+    public class maze
     {
         private const int border = 1;
-        private const int no_border = 0;
+        private const int noBorder = 0;
         private int width, height;
+        private int[,] Maze;
+        private Random randomNumber;
 
 
 
-
-        public maze(int w,int h)
+        public maze(int w, int h)
         {
             width = w;
             height = h;
```

Comment:
once the height and width are set, can these be changed? if not consider making them readonly